Fixes in dev docs (#3333)
apache/logging-log4j2: src/site/antora/modules/ROOT/pages/development.adoc

`<sourceBranch>-site-<environment>`::
`<sourceBranch>-site-<environment>-out`::
Branches used to serve the staging and production websites.

> CI automatically populates `out`-suffixed ones; you are not supposed to touch them.

Reviewer: ```suggestion
CI automatically populates `out`-suffixed branch names; you are not supposed to touch them.
```

[#contributor]
== I am **not** a committer. How shall I submit a patch?

. Is this a _trivial fix_ such as a code or documentation typo?
Simply submit a pull request.

> A Changelog entry is unnecessary. Please ensure that `./mvnw verify site` succeeds.

Reviewer 2: ```suggestion
A changelog entry is unnecessary. Please ensure that `./mvnw verify site` succeeds.
```

Go ahead and push it.

. Otherwise, submit a pull request.
Ensure a changelog entry is attached and `./mvnw verify site` succeeds.
+
[TIP]
====

> We strongly advise you to spar with another maintainer first (see {logging-services-url}/support.html#discussions-maintainer[maintainer discussion channels]) before starting to code.

Reviewer: Is "spar" the right term here? I only know that in context of practice martial arts fights (and apparently it's some part of a ship). I might go with something like "discuss" here.

====

[#release-project]
== I am a PMC member. How do I make a new release?

All Maven-based Logging Services projects are parented by
{logging-services-url}/logging-parent/index.html[Logging Parent],
streamlining several project-wide processes, including making a new release.
See {logging-services-url}/logging-parent/release-instructions-project.html[its release instructions for projects].

[#release-xml-schema]
== I am a PMC member. How do I publish a new XML schema?

All Maven-based Logging Services projects are parented by
{logging-services-url}/logging-parent/index.html[Logging Parent],
streamlining several project-wide processes, including publishing XML schemas.
See {logging-services-url}/logging-parent/release-instructions-xml-schema.html[its release instructions for XML schemas].

[WARNING]
====
**Projects and XML schemas have different lifecycles!**

> A project's new release does not necessarily mean a new release of its XML schemas.

Reviewer 3: ```suggestion
A new project release does not necessarily mean a new release of its XML schemas.
```
